### PR TITLE
fix(Lystore) #MLYSTOR-678: replace "lycée" by "établissement"

### PR DESCRIPTION
**Pull request by Rseuret**
## Describe your changes
change label
## Issue ticket number and link
https://jira.support-ent.fr/browse/MLYSTOR-678
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

**Review by qpic-cgi — APPROVED** [review]

File: src/main/resources/public/template/pdf/BC.xhtml [hunk 1]
```diff
@@ -320,7 +320,7 @@
 <div class="foot">
     <div class="left">
         <p class="bigger">
-            Lieu de Livraison, personne à contacter tel : voir liste des lycées ci-jointe <br/>
+            Lieu de Livraison, personne à contacter tel : voir liste des établissements ci-jointe <br/>
```
Comment (qpic-cgi) [inline src/main/resources/public/template/pdf/BC.xhtml:323]: j'imagine que tu ne peux pas faire du i18n là ?

Comment (Rseuret) [reply src/main/resources/public/template/pdf/BC.xhtml:323]: dans un xhtml je pense pas

Comment (qpic-cgi) [reply src/main/resources/public/template/pdf/BC.xhtml:323]: https://github.com/OPEN-ENT-NG/presences/blob/dev/presences/src/main/resources/template/pdf/absence-list-recap.xhtml#L49

tout est possible 😛

Comment (qpic-cgi) [reply src/main/resources/public/template/pdf/BC.xhtml:323]: https://github.com/OPEN-ENT-NG/presences/blob/dev/common/src/main/java/fr/openent/presences/common/service/impl/ExportPDFServiceImpl.java#L215

Comment (JinDouang) [reply src/main/resources/public/template/pdf/BC.xhtml:323]: +1 
Si possible, il faudrait les internationaliser à terme 

Après ça peut se faire sur un ticket si jamais c'est tendu niveau timing @Rseuret

File: src/main/resources/public/template/pdf/BC_Struct.xhtml [hunk 2]
```diff
@@ -304,7 +304,7 @@
     <div class="foot break">
         <div class="left" >
             <p class="bigger">
-                Lieu de Livraison, personne à contacter  tel : voir liste des lycées ci-jointe <br/>
+                Lieu de Livraison, personne à contacter  tel : voir liste des établissements ci-jointe <br/>
```
Comment (JinDouang) [inline src/main/resources/public/template/pdf/BC_Struct.xhtml:307]: idem

**Pull request merged**